src/stores/linecolors.spec.js

```diff
@@ -0,0 +1,30 @@
+import assert from 'assert';
+import {LobsterStore} from '../stores';
+
+test('Store-Line-Colors', function() {
+  const data = [
+    '[ShardedClusterFixture:job0:configsvr:secondary0]',
+    '[ShardedClusterFixture:job0:mongos]',
+    '[ShardedClusterFixture:job0:configsvr:primary]',
+    '[ShardedClusterFixture:job0:shard0:primary]',
+    '[ShardedClusterFixture:job0:configsvr:secondary1]',
+    '[ShardedClusterFixture:job0:shard0:primary]',
+    '[ShardedClusterFixture:job0:mongos]',
+    '[ShardedClusterFixture:job0:shard0:primary]',
+    '[ShardedClusterFixture:job0:shard0:secondary1]'
```

Comment:
Add an extra empty string element into this array. Nit pick, but the data that comes from logkeeper looks like this.